projects/1.21/assets/arcane-abilities/arcane_abilities/lang/zh_cn.json

```
@@ -0,0 +1,152 @@
+  "tooltip.arcane_abilities.skill.m_skill_6.tooltip": "以锹挖掘时，每1级技能等级有2%概率获得短时迅捷效果。",
+  "tooltip.arcane_abilities.skill.m_skill_7": "宝藏拆包者",
+  "tooltip.arcane_abilities.skill.m_skill_7.tooltip": "以锹挖掘时，每挖掘1个方块有0.4%概率随机获得一件宝藏箱中可能出现的物品。",
+  "tooltip.arcane_abilities.skill.m_skill_8": "高级灵魂视觉",
+  "tooltip.arcane_abilities.skill.m_skill_8.tooltip": "给予玩家夜视效果。",
+  "tooltip.arcane_abilities.skill.m_skill_9": "高效挖掘",
+  "tooltip.arcane_abilities.skill.m_skill_9.tooltip": "副手持镐挖掘时，有15%概率加快方块挖掘速度。",
+  "tooltip.arcane_abilities.skill.m_skill_10": "隐形矿工",
+  "tooltip.arcane_abilities.skill.m_skill_10.tooltip": "以镐挖掘时，有10%概率给予隐身效果。",
+  "tooltip.arcane_abilities.skill.m_skill_11": "三维采矿工艺",
+  "tooltip.arcane_abilities.skill.m_skill_11.tooltip": "增强镐的能力，使得镐挖掘的范围变为3x3。",
+
+  "tooltip.arcane_abilities.gui.category.fight": "战斗",
+  "tooltip.arcane_abilities.skill.f_skill_1": "攻如烈火",
+  "tooltip.arcane_abilities.skill.f_skill_1.tooltip": "攻击成功时，有0.5%概率点燃目标。",
+  "tooltip.arcane_abilities.skill.f_skill_2": "无尽箭矢",
+  "tooltip.arcane_abilities.skill.f_skill_2.tooltip": "弓和弩不再需要箭。",
+  "tooltip.arcane_abilities.skill.f_skill_3": "震骨重拳",
+  "tooltip.arcane_abilities.skill.f_skill_3.tooltip": "攻击成功时，在一定时间内击晕敌人并增加攻击伤害。",
+  "tooltip.arcane_abilities.skill.f_skill_4": "爆炸性重击",
+  "tooltip.arcane_abilities.skill.f_skill_4.tooltip": "每级有1%概率造成爆炸，对周围敌人造成伤害。",
+  "tooltip.arcane_abilities.skill.f_skill_5": "召唤魔法石",
+  "tooltip.arcane_abilities.skill.f_skill_5.tooltip": "攻击成功时，每级有0.5%概率在玩家周围制造出魔法石，对周围敌人造成伤害。",
+  "tooltip.arcane_abilities.skill.f_skill_6": "灵魂采收",
+  "tooltip.arcane_abilities.skill.f_skill_6.tooltip": "有0.4%概率在击杀生物时掉落1个灵魂。",
+  "tooltip.arcane_abilities.skill.f_skill_7": "雷电重击",
+  "tooltip.arcane_abilities.skill.f_skill_7.tooltip": "攻击时，每1级技能有2%概率同时攻击周围敌人。",
+  "tooltip.arcane_abilities.skill.f_skill_8": "高能重击",
+  "tooltip.arcane_abilities.skill.f_skill_8.tooltip": "攻击时，每1级技能有10%概率恢复玩家半颗心的生命值。",
+  "tooltip.arcane_abilities.skill.f_skill_9": "精确重击",
+  "tooltip.arcane_abilities.skill.f_skill_9.tooltip": "增加攻击成功时造成暴击伤害的概率。",
+  "tooltip.arcane_abilities.skill.f_skill_10": "冲锋无阻",
+  "tooltip.arcane_abilities.skill.f_skill_10.tooltip": "按下【鼠标右键】时，玩家立即冲向十字准星方向的敌人。",
+  "tooltip.arcane_abilities.skill.f_skill_11": "坚韧如钢",
+  "tooltip.arcane_abilities.skill.f_skill_11.tooltip": "暂时增加玩家的防御力，减少所受伤害。",
+
+
+
+  "item.arcane_abilities.fate_beacon": "命运信标",
+  "tooltip.relics.fate_beacon.description": "遗物置于拥有者的手上时，它将指向最近的水下结构，同时可以传送到拥有者曾标记过的地点。",
+  "tooltip.relics.fate_beacon.ability.compass": "指引辉光",
+  "tooltip.relics.fate_beacon.ability.compass.predicate.overworld": "主世界",
+  "tooltip.relics.fate_beacon.ability.compass.description": "制造一个指向最近结构的指针，指针颜色由结构类型决定。依据不同颜色可分别定位到§6埋藏的宝藏§r、§3沉船§r、§1海底神殿§r和§5海底废墟§r。",
+  "tooltip.relics.fate_beacon.ability.set_beacon": "信标标记",
+  "tooltip.relics.fate_beacon.ability.set_beacon.predicate.overworld": "主世界",
+  "tooltip.relics.fate_beacon.ability.set_beacon.description": "设定“瞬时返回”的传送点。仅在主世界中有效。",

```

```suggestion
  "tooltip.relics.fate_beacon.ability.set_beacon.description": "设定«瞬时返回»的传送点。仅在主世界中有效。",
```
是为了保持格式一致，其他用双引号的能力名也需要改